tests/test_benchmarking.py

```diff
@@ -0,0 +1,44 @@
+"""Tests for config files in benchmarking/ folder."""
+import os
+import subprocess
+
+import pytest
+
+from imitation.data import types
+
+BENCHMARKING_DIR = types.parse_path("benchmarking")
+
+if not BENCHMARKING_DIR.exists():  # pragma: no cover
+    raise RuntimeError(
+        "The benchmarking/ folder has not been found. Make sure you are "
+        "running tests relative to the base imitation project folder.",
+    )
+
+
+@pytest.mark.parametrize(
+    "script_name, command_name",
+    [
+        ("train_imitation", "bc"),
+        ("train_imitation", "dagger"),
+        ("train_adversarial", "airl"),
+        ("train_adversarial", "gail"),
+    ],
+)
+def test_benchmarking_configs(tmpdir, script_name, command_name):
+    # We test that the configs using the print_config command
+    # only for the half_cheetah environment,
+    # because the print_config command is slow
+    # (takes about 5 seconds per execution).
+    # We do not test that the configs run even with the fast configs applied,
+    # because running the configs requires MuJoCo. Requiring MuJoCo to run
+    # the tests adds too much complexity.
+    config_file = f"example_{command_name}_seals_half_cheetah_best_hp_eval.json"
+    config_path = os.path.join(BENCHMARKING_DIR.stem, config_file)
+    completed_process = subprocess.run(
```

Comment:
Do we need a subprocess for this? I think you can do something like:

```
from imitation.scripts import train_imitation
train_imitation.train_imitation_ex.run(command_name='print_config')
```

This should make things faster (5 seconds I expect is mostly from spinning up a new Python interpreter) so you could maybe check all the config files.

Not sure if you'll need to load the JSON yourself and feed it in as `config_updates` or if there's an argument to `run` for that.

You might find `tests/scripts/test_scripts.py` generally instructive on how we've been testing Sacred experiments.

Reply:
Great! I can test all the configs.